db/migrate/20130709105614_add_editable_is_default_and_index_on_editable_is_default_and_name_to_spree_roles.rb

```diff
@@ -3,7 +3,6 @@ def change
     add_column :spree_roles, :editable, :boolean, :default => true
     add_column :spree_roles, :is_default, :boolean, :default => false
 
-    add_index(:spree_roles, :name)
```

Comment:
@ShubhamGupta - Looks good to me. It will be good if we can add this change to change log too. :+1: